src/lib/layout/unauthenticated.svelte

```diff
@@ -127,4 +131,50 @@
         -webkit-background-clip: text;
         -webkit-text-fill-color: transparent;
     }
+
+    .aw-eyebrow {
+        font-family: 'Source Code Pro';
+        line-height: 16px;
+        font-size: 12px;
+        letter-spacing: 0.08em;
+        font-weight: 400;
+        text-transform: uppercase;
+    }
+
+    .aw-badges {
+        --p-badges-shadow-bg-color: #f2c8d6;
+        --p-badges-shadow-border-color: #f69db7;
+        --p-badges-shadowopacity: 0.4;
+        align-self: center;
+        color: hsl(var(--color-neutral-0));
+        padding-block: 0.375rem;
+        padding-inline: 0.75rem;
+        border-radius: 0.375rem;
+        background: radial-gradient(
+            98.13% 199.7% at 98.13% 100%,
+            #fe6947 0%,
+            #fd366e 41.6%,
+            #fd366e 100%
+        );
+        -webkit-backdrop-filter: blur(2.5rem);
+        backdrop-filter: blur(2.5rem);
+        box-shadow:
+            0.1875rem 0.1875rem var(--p-badges-shadow-bg-color),
+            0.25rem 0.1875rem var(--p-badges-shadow-border-color),
+            0.1875rem 0.25rem var(--p-badges-shadow-border-color),
+            0.125rem 0.1875rem var(--p-badges-shadow-border-color),
+            0.1875rem 0.125rem var(--p-badges-shadow-border-color);
```

Comment:
Shadows might be tricky, I'd use what we have and create a task for fixing this one we do the porting to the new colors in pink

Reply:
> I'd use what we have

@ArmanNik, what do you mean?

Reply:
I tried using the [`shadow-large` class](https://pink.appwrite.io/foundations/elevation), but it didn't add much of a shadow and it looked different than the design.

Reply:
I'd personally say following the design is more important. Does using inspect on Figma (in dev mode) return a box-shadow property for you? If so, you can use that directly while Pink/Console is not updated

Reply:
> I'd personally say following the design is more important. Does using inspect on Figma (in dev mode) return a box-shadow property for you? If so, you can use that directly while Pink/Console is not updated

@TGlide the design doesn't use a box shadow. it uses extra divs and hard-coded sizes. I've been fiddling with the CSS for a few hours and made no progress so I'm thinking it might be better to leave as is for now and revisit when the website is updated. Then, we can update it in the Console. How does that sound?